recipes/libpng/all/libpng_extra.cmake

```diff
@@ -0,0 +1,5 @@
+# define additional variables of PNG
+
+# https://cmake.org/cmake/help/v3.6/module/FindPNG.html
+set(PNG_LIBRARY "${PNG_LIBRARIES}")
+set(PNG_VERSION_STRING "${PNG_VERSION}")
```

Comment:
Are these names something standardized now in CMake? Would it be relevant to include this in the CMake find generators?

Reply:
It might, but there will always be exceptions.
Please see my comment in my openssl pr.

Reply:
Could you point me to the openssl PR?

Reply:
this PR: https://github.com/conan-io/conan-center-index/pull/486

Reply:
I am still not sure about adding those variables manually here 🤔